docs/source/app_developers_guide/kubernetes_test_network.rst

```diff
@@ -132,67 +119,172 @@ information, see the
+   <https://kubernetes.io/docs/setup/minikube/>`_.
 
-      $ kubectl run hello-minikube --image=k8s.gcr.io/echoserver:1.10 --port=8080
+   #. Start Minikube's "Hello, World" test cluster, ``hello-minikube``.
 
-      $ kubectl expose deployment hello-minikube --type=NodePort
+      .. code-block:: console
 
-#. Check the list of pods.
+         $ kubectl run hello-minikube --image=k8s.gcr.io/echoserver:1.10 --port=8080
 
-   .. code-block:: console
+         $ kubectl expose deployment hello-minikube --type=NodePort
 
-      $ kubectl get pods
+   #. Check the list of pods.
 
-   After the pod is up and running, the output of this command should display a
-   pod starting with ``hello-minikube...``.
+      .. code-block:: console
 
-#. Run a ``curl`` test to the cluster.
+         $ kubectl get pods
 
-   .. code-block:: none
+      After the cluster is up and running, the output of this command should
+      display a pod starting with ``hello-minikube...``.
 
-      $ curl $(minikube service hello-minikube --url)
+   #. Run a ``curl`` test to the cluster.
 
-#. Remove the ``hello-minikube`` cluster.
+      .. code-block:: none
 
-   .. code-block:: console
+         $ curl $(minikube service hello-minikube --url)
+
+   #. Remove the ``hello-minikube`` cluster.
 
-      $ kubectl delete services hello-minikube
+      .. code-block:: console
 
-      $ kubectl delete deployment hello-minikube
+         $ kubectl delete services hello-minikube
+
+         $ kubectl delete deployment hello-minikube
 
 
 Step 3: Download the Sawtooth Configuration File
 ------------------------------------------------
 
-Download the Kubernetes configuration (kubeconfig) file for a Sawtooth network,
-`sawtooth-kubernetes-default-poet.yaml <./sawtooth-kubernetes-default-poet.yaml>`_.
+Download the Kubernetes configuration (kubeconfig) file for a Sawtooth network.
+
+* For PBFT, download
+  `sawtooth-kubernetes-default-pbft.yaml <./sawtooth-kubernetes-default-pbft.yaml>`_
 
-This kubeconfig file creates a Sawtooth network with five pods, each running a
-Sawtooth validator node. The pods are numbered from 0 to 4.
+* For PoET, download
+  `sawtooth-kubernetes-default-poet.yaml <./sawtooth-kubernetes-default-poet.yaml>`_
 
-The configuration file also specifies the container images to download (from
+The kubeconfig file creates a Sawtooth network with five pods, each running a
+Sawtooth node. It also specifies the container images to download (from
 DockerHub) and the network settings needed for the containers to communicate
 correctly.
 
 
-Step 4: Start the Sawtooth Cluster
+Step 4: (PBFT Only) Configure Keys for the Kubernetes Pods
+----------------------------------------------------------
+
+.. important::
+
+   Skip this step if you are using PoET consensus.
+
+For a network using PBFT consensus, this step generates public and private
+validator keys for each pod in the network, then creates a Kubernetes ConfigMap
+so that the Sawtooth network can use these keys.
```

Comment:
Not a necessity (and maybe TMI for this guide), but you could maybe explain why this step is necessary for PBFT (members list needs to be known before genesis).

Reply:
I like it.  I just pushed a fixup commit with this change (and a bunch of typo fixes and wording tweaks elsewhere in the section).